clang/include/clang/Basic/arm_sve.td

```diff
@@ -1859,19 +1859,28 @@ def SVBGRP   : SInst<"svbgrp[_{d}]",   "ddd", "UcUsUiUl", MergeNone, "aarch64_sv
 def SVBGRP_N : SInst<"svbgrp[_n_{d}]", "dda", "UcUsUiUl", MergeNone, "aarch64_sve_bgrp_x">;
 }
 
+let TargetGuard = "sve2p1|sme" in {
```

Comment:
```suggestion
let TargetGuard = "sve2p1|sme2" in {
```
Should this be sme2?

Reply:
Accroding to https://developer.arm.com/documentation/ddi0602/2022-09/SVE-Instructions/PSEL--Predicate-select-between-predicate-register-or-all-false-?lang=en, it is "sve2p1" and "sme" both enabled, I think now it is `sve2p1,sme` should work.